minifi-docs/src/main/markdown/System_Admin_Guide.md

```diff
@@ -650,6 +650,16 @@ NiFi Properties Overrides:
   nifi.database.directory: ./database_repository_override
 ```
 
+# Running as a Windows Service
```

Comment:
A Windows service appears to run with a working directory of `C:\Windows\System32` and this causes problems when the directories in the configuration are relative paths. Setting them to absolute paths lets the service find the files it wants. Not my preferred solution but is cleaner than others I tried. If anyone has any other ideas let me know.